helper/tests/helper-tests-allavro/src/test/java/com/linkedin/avroutil1/compatibility/AvroWireFormatCompatibilityTest.java

```diff
@@ -98,4 +100,39 @@ public void testCompatibleJsonParsingOfWrongNumericLiterals() throws Exception {
     Assert.assertEquals(deserialized2.get(schema.getField("floatField").pos()), 3.0f);
     Assert.assertEquals(deserialized2.get(schema.getField("doubleField").pos()), 4.0d);
   }
+
+  @Test
+  public void testDefaultRecordsUnionFieldMustBeFirstElement() throws Exception {
```

Comment:
we want to call this method demonstrate*() instead of test*() ?

Reply:
yup will update